erpcgen/src/templates/c_coders.template

```diff
@@ -60,23 +60,26 @@ if (({$info.sizeTemp} <= {$info.maxSize}) && ({$info.dataTemp} != NULL))
 {% else %}
 {%   set indent = "" >%}
 {% endif %}
+{$indent}if ({$info.sizeTemp} > 0)
```

Comment:
I am not so familiar here currently, dose {$info.name} is going to get free? if yes so need to put NULL in it if you decided not to malloc into it. (if this is really the case, maybe it is cleaner just to call malloc, sorry).

Reply:
Well depends. In some cases erpc is doing freeing memory. In some the users needs to do it. Actually maybe we need to do it as we are not allocating everytime as before.